sycl/source/detail/error_handling/enqueue_kernel.cpp

```diff
@@ -210,14 +210,71 @@ bool handleInvalidWorkGroupSize(const device_impl &DeviceImpl, pi_kernel Kernel,
       "PI backend failed. PI backend returns: " + codeToString(Error), Error);
 }
 
+bool handleInvalidWorkItemSize(const device_impl &DeviceImpl,
+                               const NDRDescT &NDRDesc) {
+
+  const plugin &Plugin = DeviceImpl.getPlugin();
+  RT::PiDevice Device = DeviceImpl.getHandleRef();
+
+  size_t MaxWISize[] = {0, 0, 0};
+
+  Plugin.call<PiApiKind::piDeviceGetInfo>(Device, PI_DEVICE_MAX_WORK_ITEM_SIZE,
+                                          sizeof(MaxWISize), &MaxWISize,
+                                          nullptr);
+  for (int i = 0; i < NDRDesc.Dims; i++) {
```

Comment:
```suggestion
  for (int I = 0; i < NDRDesc.Dims; i++) {
```

Reply:
What is this change for?

Reply:
This change is for compliance of your code to coding style in current project. All variables should be written in upper camel case.